personal_assistant/contacts/templates/contacts/add_contact.html

```diff
@@ -0,0 +1,44 @@
+{% extends "pa_core/home.html" %} {% block title%}
```

Comment:
Дещо скоротиться файл, якщо унаслідуватися від іншого:
```suggestion
{% extends "pa_core/form.html" %} {% block title%}
```
Правда там немає кнопки **Reset** але вона - це вже пережитки минулого тож можна обійтися і без неї.